### ISSUE TITLE
Labeling noise?

### ISSUE THREAD

**Issue by ryohachiuma** [OpenNLPLab/AVSBench#24]
Dear authors,

I have a minor question about the annotation in the dataset.
When I debugged the code, I found that some inconsistencies existed between the ground-truth annotation and the audio label in the metadata.

For example, in the video ID `2Sg6Yq5S9Bo_0_5000`, the annotated audio object is `guitar`, and I manually checked the RGB-labeled image to make sure that the guitar is only annotated.
However, when I loaded that image, I found that a few pixels were annotated with different labels. (same as index-labeled image)

```

>>> pil = Image.open('0.png').convert('RGB')
>>> d = np.array(pil)
>>> np.unique(data.reshape(-1, data.shape[2]), return_counts=True, axis=0)
(array([[  0,   0,   0],
       [128,   0,   0],
       [128,  64,  64]], dtype=uint8), array([230097,      8,  61495]))
```

`[128, 0, 0]` corresponds to the category `baby`.  However, there is no baby in either the image or audio.
Not only this video, but some videos have these inconsistent annotations.

Please correct me if I misunderstood something!

Thank you in advance.

**Comment by cyh-0** [comment]
Hi Ryo,

I also found several cases that contain noisy labels similar to yours. I reckon that may be caused by the annotator (i.e., sometimes they accidentally click on the image.). It seems to be normal for a large-scale dataset. Anyway, I think we can still manually filter out part of these points by thresholding GT based on return_counts for the training set.

Cheers,
yh

**Comment by ryohachiuma** [comment]
@cyh-0 Thank you for your comment!
I understand the situation.

Best regards,
Ryo
